src/backend/distributed/planner/distributed_planner.c

```diff
@@ -205,6 +211,22 @@ distributed_planner(Query *parse, int cursorOptions, ParamListInfo boundParams)
 }
 
 
+/*
+ * ExtractRangeTblList is a wrapper around ExtractRangeTableEntryWalker.
+ * The function travers the input query and returns all the range table
+ * entries that is in the query tree.
+ */
+List *
+ExtractRangeTblList(Query *query)
```

Comment:
Maybe `ExtractRangeTblEntryList`, since it's not that much longer and there are is also `RangeTblFunction` and `RangeTblRef`.